homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -232,6 +260,106 @@ class XiaomiMiioSensorDescription(SensorEntityDescription):
         state_class=STATE_CLASS_TOTAL_INCREASING,
         entity_registry_enabled_default=False,
     ),
+    ATTR_DND_START: XiaomiMiioSensorDescription(
+        key=ATTR_DND_START.split(ATTR_DND)[1],
```

Comment:
If we move to 1 container per platform then the .split is not needed. As we can store the key in the constant without overlaps. The reason split is needed is bc we only need the word `start` as key/attribute. Instead of `do_not_disturb_start`